packages/juce-blueprint/src/lib/BlueprintRenderer.ts

```diff
@@ -19,30 +23,26 @@ const HostConfig = {
    *  Really only using this and `getChildHostContext` for enforcing nesting
    *  constraints, such as that raw text content must be a child of a <Text>
    *  element.
-   *
-   *  @param {Container} rootContainerInstance
    */
-  getRootHostContext(rootContainerInstance) {
+  getRootHostContext(rootContainerInstance: ViewInstance): HostContext {
```

Comment:
Hopefully all the typings here are sensible ... I wasn't totally sure. TS seems happy.

Reply:
Definitely seems like soemthing we'll want to type against react-reconciler if we can. Otherwise we might want to just `any` all the places we touch the reconciler and try to establish our type inference above it